aspnetcore/migration/mvc.md

```diff
@@ -41,19 +41,13 @@ Create a new *empty* ASP.NET Core web app with the same name as the previous pro
 
 ## Configure the site to use MVC
 
-* Install the `Microsoft.AspNetCore.Mvc` and `Microsoft.AspNetCore.StaticFiles` NuGet packages.
+* By default, the ASP.NET Core metapackage is added to the project, called `Microsoft.AspNetCore.All`. This package contains packages like `Microsoft.AspNetCore.Mvc` and `Microsoft.AspNetCore.StaticFiles`.
```

Comment:
This metapackage is ONLY used when targeting .NET Core. If targeting .NET Framework, package references need to be listed individually in the *.csproj file.